### DISCUSSION TITLE
A fix for the 500 Server Error problem when installing SnipeIT on Synology (Docker container)

### DISCUSSION THREAD

**Discussion by danielrosehill** [snipe/snipe-it#14424] [General]
I followed the Marius Hosting instructions to install Snipe IT on a Synology NAS as a Docker container (using Portainer).

But my Pre Flight Check kept chocking on the database creation.

I figured out a very simple fix!

Following the tutorial will result in the creation of two containers:

![image](https://github.com/snipe/snipe-it/assets/40233203/9271e8fd-38a6-4a33-9b8d-fd376eaeb097)

You simply need to manually add a 'link' between the two so that the main container can talk to the DB server. 

First, power down the containers.

Then just add a link from SNIPE-IT to SNIPE-IT-DB:

![image](https://github.com/snipe/snipe-it/assets/40233203/d6590d3f-6a17-4f98-bde1-b105dce99ff3)

You can now restart the containers and you should be able to finish the installation including the database creation without running into the server error!

**Comment by Suppo94** [comment]
thank you for sharing the solution, however, no matter what I do, I link the 2 containers, but when I go to the main "Containers" screen and back into the container settings, it doesn't show the link I just created, and that happens on both containers, even though when I create the link, and click save, it shows changes saved, but still, doesn't really save
<img width="1792" alt="Screenshot 2024-09-23 at 7 46 32 PM" src="https://github.com/user-attachments/assets/ca7fcb16-9d44-4340-bacf-882b99d7b80d">

**Reply by ThichCaRotCEO** [reply]
<img width="731" height="246" alt="image" src="https://github.com/user-attachments/assets/2877b4d1-eb4b-4342-adeb-3a6b2d723028" />
You can try service apache2 restart as in my Console told that. At least it worked for me
